tests/CMakeLists.txt

```diff
@@ -262,3 +262,7 @@ endif()
 
 list(APPEND CATCH_WARNING_TARGETS SelfTest)
 set(CATCH_WARNING_TARGETS ${CATCH_WARNING_TARGETS} PARENT_SCOPE)
+
+if(CATCH_BUILD_SURROGATES)
+    add_subdirectory(SurrogateCpps)
+endif()
```

Comment:
The option should also have a top level definition, so that various CMake GUIs pick it up better. (I'll take care of this)